src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -3,9 +3,10 @@
 
   <!-- Official Version -->
   <PropertyGroup>
-    <MajorVersion>2</MajorVersion>
-    <MinorVersion>3</MinorVersion>
+    <MajorVersion>3</MajorVersion>
+    <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
+    <PreviewVersion>-preview</PreviewVersion>
```

Comment:
We should only change the version of `Microsoft.FeatureManagement` package first. Once that has been released to NuGet, we can update the version in `Microsoft.FeatureManagement.AspNetCore`.

Reply:
Thanks, updated.